dali/operators/image/color/color_twist.cu

```diff
@@ -32,6 +32,7 @@ DALI_REGISTER_OPERATOR(Saturation, ColorTwistGpu, GPU);
 DALI_REGISTER_OPERATOR(ColorTwist, ColorTwistGpu, GPU);
 
 bool ColorTwistGpu::SetupImpl(std::vector<OutputDesc> &output_desc, const DeviceWorkspace &ws) {
+  KMgrResize(num_threads_, max_batch_size_);
```

Comment:
Why max and not current batch size? (I know it's probably more stable this way).